cdci_data_analysis/analysis/products.py

```diff
@@ -334,35 +313,42 @@ def write(self, file_name=None, overwrite=True, file_dir=None):
         else:
             file_path = self.file_path.get_file_path(file_name=file_name, file_dir=file_dir)
 
-
-        self.data.write_fits_file(file_path, overwrite=overwrite)
-
-    def add_url_to_fits_file(self,par_dict,url='',use_primary=True,add_query_dict=True):
+        # turns out the way astropy writes some structures really overloads some nfs mounts.
+        # this happened on UNIGE site after update. 
+        # workaround is made here and should be configurable
+        logger.debug("\033[31mbefore %s.write(file_name=%s) as %s\033[0m", self, file_name, file_path)
+
+        with tempfile.NamedTemporaryFile(delete=False) as f:
+            self.data.write_fits_file(f, overwrite=overwrite)
+            f.close()
```

Comment:
since it's inside a `with` block, this should not be necessary...

Reply:
> since it's inside a `with` block, this should not be necessary...

see next time, I need it to be closed before moving.

Reply:
oh yes, didn't properly see that

Reply:
maybe you are comparing with `open` context which works differently. 
Here really we do not really need context exit, but that's one of the nicer ways  for `tempfile` to make a named file.

Reply:
indeed, I thought it would basically work in the same way